business_objects/monitor.py

```diff
@@ -0,0 +1,424 @@
+from typing import Any, List
+from . import general
+from .. import enums
+
+
+def get_all_tasks(project_id: str = None, only_running: bool = False) -> List[Any]:
+    query = f""" 
+    SELECT tasks.*, orga.name organization_name
+    FROM (
+        ({__select_running_information_source_payloads(project_id, only_running)})
+        UNION
+        ({__select_running_attribute_calculation_tasks(project_id, only_running)})
+        UNION
+        ({__select_running_tokenization_tasks(project_id, only_running)})
+        UNION
+        ({__select_running_embedding_tasks(project_id, only_running)})
+        UNION
+        ({__select_running_weak_supervision_tasks(project_id, only_running)})
+        UNION
+        ({__select_running_upload_tasks(project_id, only_running)})
+    ) tasks
+    JOIN project p
+    ON p.id = tasks.project_id
+    JOIN organization orga
+    ON orga.id = p.organization_id
+    """
+
+    return general.execute_all(query)
+
+
+def cancel_all_running_tasks(project_id: str = None):
+    set_information_source_payloads_to_failed(project_id)
+    set_attribute_calculation_to_failed(project_id)
+    set_embedding_to_failed(project_id)
+    set_record_tokenization_task_to_failed(project_id)
+    set_upload_task_to_failed(project_id)
+    set_weak_supervision_to_failed(project_id)
+    general.commit()
+
+
+def set_information_source_payloads_to_failed(
+    project_id: str = None, payload_id: str = None, with_commit: bool = False
+) -> None:
+    query = f"""
+    UPDATE information_source_payload
+    SET state = '{enums.PayloadState.FAILED.value}'
+    WHERE state = '{enums.PayloadState.STARTED.value}'
+    """
+
+    if project_id:
+        query = (
+            query
+            + f"""
+        AND project_id = '{project_id}'
+        """
+        )
```

Comment:
a bit hard to read imo. I'd go with:
```suggestion
   if project_id:
        query+=f" AND project_id = '{project_id}'"
```

If we want a linebreak \n so we don't need to write the linebreak in the triple quotes.

Reply:
same for the others as well

Reply:
done

Reply:
The line breaks came from the formatter I guess IMO we don't need them